### PR TITLE
Enable unit test workflow on Arc

### PR DESCRIPTION
**Pull request by chtanch**
## Description
Enable github workflow for llm unit test on Arc.  An inference test for transformers API is also implemented to verify that the workflow runs correctly. The inference test queries models with a simple question and checks for the correct answer.

### CODE REVIEW

File: .github/workflows/llm_unit_tests.yml [hunk 1]
```diff
@@ -186,3 +186,78 @@ jobs:
...
+        with:
+          extra-dependency: "xpu"
+
+      - name: Test installed xpu version
+        shell: bash
+        run: |
+          source /opt/intel/oneapi/setvars.sh
+          bash python/llm/test/run-llm-install-tests.sh
+
+      - name: Download LLMs
```
Comment (hkvision) [inline .github/workflows/llm_unit_tests.yml:238]: I don't think we need to download models here?

File: python/llm/test/run-llm-inference-tests-gpu.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+if [ -z "$THREAD_NUM" ]; then
+  THREAD_NUM=2
+fi
+export OMP_NUM_THREADS=$THREAD_NUM
+pytest ${LLM_INFERENCE_TEST_DIR} -v -s 
+
+now=$(date "+%s")
+time=$((now-start))
+
+echo "Bigdl-llm tests finished"
```
Comment (hkvision) [inline python/llm/test/run-llm-inference-tests-gpu.sh:25]: Bigdl-llm gpu tests finished

File: python/llm/test/inference_gpu/test_transformers_api.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    import intel_extension_for_pytorch as ipex
+
+@pytest.mark.parametrize('prompt, answer', [
+    ('What is the capital of France?\n\n','Paris')
+    ])
+@pytest.mark.parametrize('Model, Tokenizer, model_path',[
+    (AutoModelForCausalLM, LlamaTokenizer, os.environ.get('LLAMA2_7B_ORIGIN_PATH')),
+    (AutoModel, AutoTokenizer, os.environ.get('CHATGLM2_6B_ORIGIN_PATH')),
+    (AutoModelForCausalLM, AutoTokenizer, os.environ.get('FALCON_7B_ORIGIN_PATH')),
+    ])
```
Comment (WeiguangHan) [inline python/llm/test/inference_gpu/test_transformers_api.py:36]: I think we may have missed MPT_7B_ORIGIN_PATH here?

Comment (hkvision) [reply python/llm/test/inference_gpu/test_transformers_api.py:36]: let's add it in another PR

**Review by hkvision — APPROVED** [review]

File: python/llm/test/inference_gpu/test_transformers_api.py (hunk 3 above)
Comment (chtanch) [reply python/llm/test/inference_gpu/test_transformers_api.py:36]: Thanks, will include it in another PR

**Pull request merged**